src/components/application_manager/test/commands/hmi/simple_response_from_hmi_test.cc

```diff
@@ -318,6 +320,32 @@ TEST_F(OtherResponseFromHMICommandsTest, VIIsReadyResponse_Run_SUCCESS) {
   command->Run();
 }
 
+TEST(NaviIsReadyResponseTest, NaviIsReadyResponse_Run_SUCCESS) {
+  application_manager_test::MockApplicationManager mock_app_manager;
```

Comment:
@dcherniev I would propose you to create a separate test file for `NaviIsReadyResponse`. Also there should be one more test case related to value of `(*command_msg)[strings::msg_params][strings::available] = false` Consider using one constant for this value and for the expectation too.